api/overview/azure/app-auth-migration.md

```diff
@@ -130,5 +130,7 @@ var accessToken = await tokenProvider.GetAccessTokenAsync(ResourceId);
 
 ```csharp
 var tokenCredential = new DefaultAzureCredential();
-var accessToken = await tokenCredential.GetTokenAsync(ResourceId);
-```
\ No newline at end of file
+var accessToken = await tokenCredential.GetTokenAsync(
+    new TokenRequestContext(scopes: new string[] { "your scope here" }) { }
```

Comment:
I wonder if we should actually help users by showing them the genaral way a ResourceId can  be turned into a scope. So perhaps something like this
```c#
var accessToken = await tokenCredential.GetTokenAsync(
    new TokenRequestContext(scopes: new string[] { ResourceId + "./default" }) { }
```
I think if someone is really trying to migrate from track 1 which uses AAD V1 (reesource strings) to track 2 which uses AAD V2 (scopes) they'll probably need this information as well. @nonik0 Do you have thoughts on this?

Reply:
It would be helpful to explain what is meant by "scope" and how one finds said scope.